src/openrct2-ui/windows/Staff.cpp

```diff
@@ -290,6 +293,8 @@ static constexpr const uint32_t window_staff_page_enabled_widgets[] = {
 
 static uint8_t _availableCostumes[ENTERTAINER_COSTUME_COUNT];
 
+static int32_t _staffPatrolAreaPaintValue = -1;
```

Comment:
It'd be easier to read the code if that was an `enum class` with modes explicitly named. Right now it's not immediately obvious what each of the values means and which are supported without referring to other uses.

Reply:
I agree